app-info/eos-extra.xml

```diff
@@ -340,15 +333,8 @@
     <bundle type="flatpak"></bundle>
   </component>
   <component type="desktop" merge="append">
-    <id>org.kde.Ktuberling.desktop</id>
-    <categories>
-      <category>Family</category>
-    </categories>
-  </component>
-  <component type="desktop" merge="append">
-    <id>org.kde.Marble.desktop</id>
```

Comment:
Here is the reference for Marble having been removed: https://phabricator.endlessm.com/T2791#233028